docs/operator-manual/project.yaml

```diff
@@ -30,6 +30,10 @@ spec:
   - group: ''
     kind: NetworkPolicy
 
+  # Enables namespace orphaned resource monitoring.
+  orphanedResources:
```

Comment:
is this the correct place for this setting?

Reply:
Yes, as we discussed offline orphaned resource monitoring settings and namespace blacklisted settings are related. Project namespace blacklisted resources should not be considered orphaned since such resources are usually managed by system admins and should not cause app warning conditions.